modules/electrophysiology_browser/help/view_session.md

```diff
@@ -0,0 +1,15 @@
+# View Session
+
+This page displays all available information collected for an individual recording session.
+
+Summary information about the candidate, the recording session and the hardware used is displayed.
+
+Recording files can be downloaded individually or as a compressed folder (*All Files* download):
```

Comment:
```suggestion
File packages can be downloaded containing just the recording signal, the events, or other metadata for this session - or all files compressed together (*All Files* download).  These file packages are organized according to the EEG-BIDS standard, and available for download as follows:
```

This suggestion/change because: the _Recordings_ can't be downloaded "individually", it's the different *types* of files that are downloadable as packages. 

Karim could you take another pass to look closer at the following? 
- e.g I think "All Files" should be in the bulleted list (ordered according to the buttons you see) 
- I'm glad you added link for BIDS; instead, let's link the EEG BIDS paper here, the URL is: https://www.nature.com/articles/s41597-019-0104-8

Reply:
I added "All Files" as part of the list. For the EEG-BIDS link, I created a footer to keep things neat in case more content was added to the file in the future.